ocs_ci/helpers/helpers.py

```diff
@@ -115,6 +115,36 @@ def wait_for_resource_state(resource, state, timeout=60):
     logger.info(f"{resource.kind} {resource.name} reached state {state}")
 
 
+def update_container_with_proxy_env(job_pod_dict):
```

Comment:
Maybe move to some proxy-helpers module as it's not that general function to be in just helpers.py

Reply:
Moved to new module `ocs_ci.helpers.proxy` in 8acb403.